runtime/doc/support.txt

```diff
@@ -11,15 +11,16 @@ Support                                                              *support*
 ==============================================================================
 Supported platforms					 *supported-platforms*
 
-`System`          `Tier`      `Versions`                  `Tested versions`
-Linux            1      >= 2.6.32, glibc >= 2.12     Ubuntu 24.04
-macOS (Intel)    1      >= 11                        macOS 13
-macOS (M1)       1      >= 11                        macOS 15
-Windows 64-bit   1      >= Windows 10 Version 1809   Windows Server 2022
-FreeBSD          1      >= 10                        FreeBSD 14
-OpenBSD          2      >= 7
-MinGW            2      MinGW-w64
-Windows 64-bit   3      < Windows 10 Version 1809
+`System`            `Tier`      `Versions`                  `Tested versions`
+Linux (x864_64)    1      >= 2.6.32, glibc >= 2.12     Ubuntu 24.04
+Linux (aarch64)    1      >= 2.6.32, glibc >= 2.12     Ubuntu 24.04
```

Comment:
```suggestion
Linux (x86_64)     1      >= 2.6.32, glibc >= 2.12     Ubuntu 24.04
Linux (arm64)      1      >= 2.6.32, glibc >= 2.12     Ubuntu 24.04
```

Reply:
That arch was intentional. Can you tell my _why_ we should use one over the other?

Reply:
No

Reply:
wat

Reply:
Well I'm not sure either. But GitHub calls it arm64.

Reply:
So does Debian, so let's go with that. (It's also shorter.)